cypress/features/regression/test/actionPopoverIFrameNoIFrame.feature

```diff
@@ -119,3 +156,51 @@ Feature: Action Popover component
       | Sub Menu 1 | 0          |
       | Sub Menu 2 | 1          |
       | Sub Menu 3 | 2          |
+    
+  @positive
+  Scenario Outline: Close submenu after press Enter keyboard key
+    Given I click the menu button element
+      And I press keyboard downarrow key times 2
+      And I wait 250
+      And I press leftarrow on focused element
+      And I wait 250
+      And I press keyboard downarrow key times <innerTimes>
+    When I press Enter on focused element
+    Then ActionPopover submenu is not visible
+    Examples:
+      | innerText  | innerTimes |
+      | Sub Menu 1 | 0          |
+      | Sub Menu 2 | 1          |
+      | Sub Menu 3 | 2          |
+
+  @positive
+  Scenario Outline: Close submenu after press ArrowRight keyboard key
+    Given I click the menu button element
+      And I press keyboard downarrow key times 2
+      And I wait 250
+      And I press leftarrow on focused element
+      And I wait 250
+      And I press keyboard downarrow key times <innerTimes>
+    When I press ArrowRight on focused element
+    Then ActionPopover submenu is not visible
+    Examples:
+      | innerText  | innerTimes |
```

Comment:
if you don't use `innerText` in steps remove it form `examples`

Reply:
updated